### ISSUE TITLE
Difference in member information for two South Carolina files

### ISSUE THREAD

**Issue by gitonthescene** [JeffreyBLewis/congressional-district-boundaries#6]
Hello,

I just thought I'd point out that the member data uses arrays for `name`, `party` and `district` in both `South_Carolina_104_to_107.geojson` and `South_Carolina_108_to_112.geojson` whereas other files have them simply as strings.  These were the only two files I found like this.  FWIW, it's not critical to me to have this consistent, I just thought you might want to know.

Regards,

```
$ cat South_Carolina_1_to_2.geojson | jq -r '.features[].properties | .member | to_entries[0]' | head -n 15 
{
  "key": "1",
  "value": {
    "8702": {
      "party": "Pro-Administration",
      "name": "Smith, William Loughton",
      "district": "1"
    }
  }
}
{
  "key": "1",
  "value": {
    "1260": {
      "party": "Anti-Administration",
$ cat South_Carolina_104_to_107.geojson | jq -r '.features[].properties | .member | to_entries[0]' | head -n 15
{
  "key": "104",
  "value": {
    "29565": {
      "party": [
        "Republican"
      ],
      "name": [
        "Sanford, Marshall \"Mark\""
      ],
      "district": [
        "1"
      ]
    }
  }
$
```

**Comment by JeffreyBLewis** [comment]
Thank you for reporting this.

**Comment by JeffreyBLewis** [comment]
Fixed in #8

**Comment by gitonthescene** [comment]
Super.  Thanks!
